service/controller/v1/resource/persistentvolume/update.go

```diff
@@ -92,27 +102,30 @@ func (r *Resource) ApplyUpdateChange(ctx context.Context, obj, updateState inter
 			return nil
 		}
 
-		if err := r.k8sClient.Batch().Jobs("kube-system").Delete(cleanupJob.Name, &metav1.DeleteOptions{}); err != nil {
+		err = r.k8sClient.Batch().Jobs("kube-system").Delete(cleanupJob.Name, &metav1.DeleteOptions{})
```

Comment:
```suggestion
		err = r.k8sClient.Batch().Jobs(metav1.NamespaceSystem).Delete(cleanupJob.Name, &metav1.DeleteOptions{})
```

Minor but I like using the namespace constants for stuff like this.